src/dropdown.ts

```diff
@@ -945,19 +953,16 @@ export default class GlideCoreDropdown
           </div>
 
           <div
-            aria-labelledby=${this.filterable || this.isFilterable
```

Comment:
As far as I can tell, there's no benefit to having these here _and_ below. And they have to be on the `role="listbox"` below.